### PR TITLE
E2e tests mpe invitation

### LINKED ISSUE #353: User receives MPE room invitation client
# Client implem of the User receives MPE room invitation

- [x] add a listener for the `MPE_RECEIVED_INVITATION` inside the `appMusicPlaylistsMachine`, in handler display an invitation toast thanks to the given MpeRoomSummary. On toast click triggers an `MPE_JOIN_ROOM` with the given `MpeRoomSummary`
- [x] Disable MpeRoomView playlist edit CTA depending on the `isOpen` and `isOpenOnlyInvitedUsersCanEdit` settings
- [x] e2e test mpe creator invite user receives and accept from toast
- [x] client integ tests

Below will be done during mpe room search engine implem ( other issue )
- [ ] Update the mpe room search engine list element to display specific icon depending the MpeRoomSummary props values, invited private etc etc ( such as for the mtv )

### PR DESCRIPTION
**Pull request by prastoin**
close #353

### CODE REVIEW

**Comment by prastoin** [comment]
Should review this PR before https://github.com/AdonisEnProvence/MusicRoom/pull/382

File: packages/client/e2e/mpe/test-d.spec.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    const { roomName } = await createMpeRoom({
+        page: creatorUserA,
+    });
+
+    //Creator invites userB
+    const inviteUserButton = creatorUserA.locator(
+        `css=[data-testid="mpe-invite-user-button"]`,
+    );
+    expect(inviteUserButton).toBeTruthy();
```
Comment (Devessier) [inline packages/client/e2e/mpe/test-d.spec.ts:53]: I know where this habit comes from 👀 React Native Testing Library!

File: packages/client/e2e/mpe/test-d.spec.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    const hasBeenInvitedIcon = creatorUserA.locator(
+        `css=[data-testid="${userBNickname}-user-card"] [aria-label="Has been invited"]`,
+    );
+    await expect(hasBeenInvitedIcon).toBeVisible();
+
+    //UserB has hit the toast
+    const joinRoomButton = joiningUserB.locator(`text="JOIN"`);
+    await expect(joinRoomButton).toBeVisible();
+    await expect(joinRoomButton).toBeEnabled();
```
Comment (Devessier) [inline packages/client/e2e/mpe/test-d.spec.ts:86]: I think we are going to need to write either a custom assertion, or a simple function, that checks whether an element is enabled through `aria-disabled` attribute.

File: packages/client/e2e/mpe/test-d.spec.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+    await joinRoomButton.click();
+
+    await expect(
+        getAddTrackButton({
+            page: joiningUserB,
+        }),
+    ).not.toHaveAttribute('aria-disabled', 'true');
```
Comment (Devessier) [inline packages/client/e2e/mpe/test-d.spec.ts:94]: A function that does this assertion.

**Review by Devessier — APPROVED** [review]
Nice!
We can merge.
I wrote some comments about our E2E tests.

**Pull request merged**